dist/initramfs/initramfs-tools/scripts/elastio-snap

```diff
@@ -46,7 +46,9 @@ if [ -n "$rbd" ]; then
 
     echo "elastio-sanp: mounting $rbd as $ROOTFSTYPE" > /dev/kmsg
     blockdev --setro $rbd
-    mount -t $ROOTFSTYPE -o ro $rbd /etc/elastio/dla/mnt > /dev/kmsg
+    if ! mount -t $ROOTFSTYPE -o ro $rbd /etc/elastio/dla/mnt > /dev/kmsg ; then
+        echo "elastio-sanp: error: cannot reload tracking data: root cannot be mount as readonly" > /dev/kmsg
+    fi
```

Comment:
May be use such:
```
    if mount -t $ROOTFSTYPE -o ro $rbd /etc/elastio/dla/mnt > /dev/kmsg ; then
        udevadm settle

        if [ -x /sbin/elastio_reload ]; then
            /sbin/elastio_reload
        else
            echo "elastio-sanp: error: cannot reload tracking data: missing /sbin/elastio_reload" > /dev/kmsg
        fi

        umount -f /etc/elastio/dla/mnt > /dev/kmsg
    else
        echo "elastio-sanp: error: cannot reload tracking data: root cannot be mount as readonly" > /dev/kmsg
    fi

```

Reply:
@kgermanov https://github.com/elastio/elastio-snap/pull/192 has been merged. Please apply your suggestion from the comment above. It seems to be logical improvement.

Reply:
Done